WEB-INF/resources/sv.lang.php

```diff
@@ -32,392 +32,382 @@
 $i18n_language = 'Svenska';
 $i18n_months = array('Januari', 'Februari', 'Mars', 'April', 'Maj', 'Juni', 'Juli', 'Augusti', 'September', 'Oktober', 'November', 'December');
 $i18n_weekdays = array('Söndag', 'Måndag', 'Tisdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lördag');
-$i18n_weekdays_short = array('sön', 'mån', 'tis', 'ons', 'tor', 'fr', 'lör');
+$i18n_weekdays_short = array('sö', 'må', 'ti', 'on', 'to', 'fr', 'lö');
 // format mm/dd
-$i18n_holidays = array('01/01', '01/06', '04/10', '04/12', '04/13', '05/01', '05/21', '05/31', '06/06', '06/20', '10/31', '11/01', '12/25', '12/26');
+$i18n_holidays = array('01/01', '01/06', '05/01', '06/06', '12/25', '12/26'); // OBS! Endast fasta helgdagar, då rörliga datum måste uppdateras varje år.
```

Comment:
In Sweden we have "moving holidays", where it can occur for example "on the third Thursday of April". 
These would have to be updated every year. 

In this commit only the holidays on fixed dates are included.